episodes/superspreading-simulate.Rmd

```diff
@@ -0,0 +1,847 @@
+
+:::::::::::::::::::::::::
+
+:::::::::::::::::::::::::: challenge
+
+### From a distribution of secondary cases
+
+[Christian Althaus, 2015](https://www.thelancet.com/journals/laninf/article/PIIS1473-3099(15)70135-0/fulltext) reused data published by [Faye et al., 2015 (Figure 2)](https://www.thelancet.com/journals/laninf/article/PIIS1473-3099(14)71075-8/fulltext#gr2) on the transmission tree on Ebola virus disease in Conakry, Guinea, 2014.
+
+Using the data under the **hint** tab, estimate the offspring distribution from the distribution of secondary cases. Then estimate the large outbreak potential from this data.
+
+::::::::::: hint
+
+Code with the transmission tree data written by [Christian Althaus, 2015](https://www.thelancet.com/journals/laninf/article/PIIS1473-3099(15)70135-0/fulltext):
+
+```{r,message=FALSE,warning=FALSE}
+# Number of individuals in the trees
+n <- 152
+# Number of secondary cases for all individuals
+c1 <- c(1, 2, 2, 5, 14, 1, 4, 4, 1, 3, 3, 8, 2, 1, 1,
+        4, 9, 9, 1, 1, 17, 2, 1, 1, 1, 4, 3, 3, 4, 2,
+        5, 1, 2, 2, 1, 9, 1, 3, 1, 2, 1, 1, 2)
+c0 <- c(c1, rep(0, n - length(c1)))
+
+c0 %>%
+  enframe() %>%
+  ggplot(aes(value)) +
+  geom_histogram()
+```
+
+:::::::::::
+
+::::::::::: solution
+
+<!-- [pending to complete] -->
+
+Remarkable, even with R0 less than 1
+with potentially explosive outbreaks. The observed variation in individual infectiousness in Ebola
+means that although the probability of extinction is high, new index cases also have the potential for explosive regrowth of the epidemic.
+
+:::::::::::
+
+:::::::::::::::::::::::::::::
+
+:::::::::::::::::::: challenge
+
+### Probability to contain an epidemic
+
+Calculate probabilities for Ebola of a large outbreak and to contain an epidemic. Use the appropriate package to access delay data from previous outbreaks.
+
+:::::::::::::::::: hint
+
+With `{epiparameter}` you can access and use delay distributions from previous outbreaks.
+
+:::::::::::::::::::::
+
+::::::::::::::::::: hint
+
+### Epidemic Risk assessment accounting for superspreading
+
+With `{superspreading}`, you can get numerical solutions to processes that `{epichains}` solve using branching processes. We invite you to read the `{superspreading}` vignette on [Epidemic risk](https://epiverse-trace.github.io/superspreading/articles/epidemic_risk.html) and respond to:
+
+- What is the probability of a pathogen causing a large outbreak?
+- What is the probability of an outbreak becoming extinct?
+- What is the probability of containing the ongoing outbreak?
+
+Check how these estimates vary non-linearly with respect to the mean reproduction number $R$ and dispersion $k$ of a given disease.
+
+::::::::::::::::::
+
+::::::::::::::::::::: hint
+
+Here is an example with the MERS offspring distribution estimates:
+
+```{r,message=FALSE,warning=FALSE}
+library(superspreading)
+```
+
+```{r}
+# estimate probability to contain
+probability_contain(
+  R = mers_offspring["mean"],
```

Comment:
Do you mean Ebola (see comment above)? Although MERS also fine as a case study

Reply:
removed but placed hidden for further evaluation in https://github.com/epiverse-trace/tutorials-middle/commit/60ceb9f801222656f8bc3ecaf7a72132096e3256